Src/library/memory_analyzer/ma_draw_history.e

```diff
@@ -15,6 +15,12 @@ inherit
 		export
 			{NONE} all
 		end
+
```

Comment:
Should we pass this as argument to the creation procedure instead? Like it is done elsewere?